packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,72 @@
+# Specify analysis options.
+
+# This file is the .analysis_options file used by Flutter editors, such as
+# Atom. It is very similar to flutter_tools/flutter_analysis_options; the only
+# difference (currently) is the public_member_api_docs option, which triggers
+# too many messages to be used in editors.
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: true
```

Comment:
Strong mode today, strong mode with the four strong_mode_* fields turned off as we have them below, and strong mode for 2.0, are all very different beasts. Given that many of our customers are coming from the JS "types shmypes" school of programming, I doubt that they'd want strong mode as we have it today, and trying to explain what strong mode means with all the flags below turned to "ignore" is difficult (I'm not sure I could even tell you what it means to be honest... we turned it on just to get a bit more type checking, but as discussed elsewhere, we should probably not turn on type checking by default).